### ISSUE TITLE
This is a copy of an existing repository

### ISSUE THREAD

**Issue by hugozap** [DistributedWeb/dweb-eos#1]
@JaredRiceSr  I noticed this is obviously a  copy of https://github.com/mafintosh/end-of-stream/blob/master/index.js however no credits/license information is present as required by the MIT license.

**Comment by JaredRiceSr** [comment]
You're correct. I spoke on this in his chat room earlier. I am not promoting this project and I'm just working on things at this moment. I have added Mafintosh to the license and we are working on the other LICENSES since it seems that this project has now been promoted.
